apps/safe-claiming-app/public/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "short_name": "safe-claiming-app",
+  "name": "$SAFE Claiming App",
+  "description": "Claim your $SAFE Tokens",
+  "iconPath": "",
+  "icons": [
```

Comment:
We are missing an icon in the manifest

Is there anything expected?

Reply:
I'm discussing it. It will most likely be the safe token icon. I'll add it when decided :)

Reply:
The safe token logo was added!